Fix to-device encryption info label

toger5 opened the pull request:
fixes: https://github.com/element-hq/element-call/issues/3201
The label was shown also without checking that we use PerParticipantE2EE. Which is a prerequisite for toDevice transport. As a result the label was shown when not desired.

toger5 commented on src/room/InCallView.tsx:
```diff
@@ -220,19 +223,21 @@ export const InCallView: FC<InCallViewProps> = ({
     rtcSession,
     RoomAndToDeviceEvents.EnabledTransportsChanged,
-    (enabled) => setShowToDeviceEncryption(enabled.to_device),
+    (enabled) => setDidFallbackToRoomKey(enabled.room),
```
This is the actual bug: I used `enabled.to_device` but of course `enabled.room` is better.

BillCarsonFr reviewed (approved):
we can't have a test?

toger5 commented:
@BillCarsonFr yes, I will add a test.

toger5 committed:
Fix to-device encryption info label
The label was shown also without checking that we use PerParticipantE2EE. Which is a prerequisite for toDevice transport. As a result the label was shown when not desired.

toger5 committed:
rename: useLiveKit -> useLivekit

toger5 committed:
Add tests

toger5 committed:
lint

toger5 committed:
make the settings naming consistent
